pages/help-center/errors/replication.mdx

```diff
@@ -60,7 +60,7 @@ need a snapshot of the database, then create one on the main instance.
 ## Multi-tenant replication is not supported [#warning-2]
 
 Memgraph supports multi-tenancy. However, multi-tenant replication is, at the
-moment, experimental. See [Experimental replication](/clustering/replication/experimental-replication) for more info.
+moment, experimental. See [System replication](/clustering/replication/system-replication) for more info.
```

Comment:
This whole section is not applicable because multi-tenancy under replication is now supported